test/hermes_test/commands/harvest/test_cff.py

```diff
@@ -64,3 +64,44 @@ def test_validate_success(valid_minimal_cff):
 
 def test_validate_fail():
     assert harvest._validate(pathlib.Path("foobar"), {}) is False
+
+
+
```

Comment:
Too many blank lines...
```suggestion


```

Reply:
Done in local commit that I forgot to push.